docs/conf.py

```diff
@@ -55,8 +55,8 @@
 autodoc_typehints = 'description'
 
 autodoc_type_aliases = {
-    'DataArrayLike': 'DataArrayLike',
-    'DatasetLike': 'DatasetLike',
+    'LabeledArray': 'LabeledArray',
+    'MetaDataMap': 'MetaDataMap',
```

Comment:
The latter refers to `Coord` or `Masks`?

Reply:
Maybe. The corresponding class in `typing` is `MutableMapping`, see definition of `MetaDataMap`.